### ISSUE TITLE
Bug in stub file for verify method

### ISSUE THREAD

**Issue by SHxKM** [Legrandin/pycryptodome#337]
The stub file for Crypto pkcs1_15.pyi has the following definition for `verify`: 

``` 
def verify(self, msg_hash: ModuleType, signature: bytes) -> None: ...
```

As explained [here](https://github.com/microsoft/pyright/issues/320#issuecomment-549166570), this doesn’t make sense nor does it line up with the docs.

**Comment by Legrandin** [comment]
Thanks for flagging this. The changes [in this branch](https://github.com/Legrandin/pycryptodome/tree/issue_337) should fix the problem.

**Comment by Legrandin** [comment]
Fixed in version 3.9.2.
